apps/opik-backend/src/main/java/com/comet/opik/domain/TraceDAO.java

```diff
@@ -897,6 +927,11 @@ private Publisher<Trace> mapToDto(Result result) {
                 .totalEstimatedCost(row.get("total_estimated_cost", BigDecimal.class).compareTo(BigDecimal.ZERO) == 0
                         ? null
                         : row.get("total_estimated_cost", BigDecimal.class))
+                .errorInfo(Optional.ofNullable(row.get("error_info", String.class))
+                        .filter(str -> !str.isBlank())
+                        .map(errorInfo -> JsonUtils.readValue(errorInfo, new TypeReference<ErrorInfo>() {
```

Comment:
For the read path, better use `getJsonNodeFromString`.

Reply:
Sounds good, the last minor thing would be to create a constant for `new TypeReference<ErrorInfo>` and place it as `static final` field in `ErrorInfo`. These objects are thread safe. You'll re-use it from everywhere instead of instantiating each time.

Reply:
Good suggestion, updated